### DISCUSSION TITLE
Animation Control

### DISCUSSION THREAD

**Discussion by DEADSMAN** [google/model-viewer#3586] [Q&A]
I want to trigger the animation in the GLB file with the help of a button after placing the object on the screen as AR. Any ideas how I can do it?

**Comment by elalish** [comment]
Trigger this method: https://modelviewer.dev/docs/#entrydocs-animation-methods-play from this event handler: https://modelviewer.dev/docs/#entrydocs-augmentedreality-events-arStatus, probably waiting for `object-placed`.
